Patches.py

```diff
@@ -2112,6 +2120,10 @@ def update_scrub_text(message, text_replacement, default_price, price, item_name
     SKULL_CHEST_BIG =  15
     HEART_CHEST_SMALL = 16
     HEART_CHEST_BIG = 17
+    if world.settings.shuffle_tcgkeys == 'vanilla':
+        item = read_rom_item(rom, 0x71)
```

Comment:
This could use a comment explaining what it's doing (changing chest game keys to be in brown chests).

Reply:
addressed